Library/Formula/swift.rb

```diff
@@ -77,20 +78,21 @@ def install
         "--build-args=-j#{ENV.make_jobs}",
         "--lldb-use-system-debugserver",
         "--install-prefix=#{prefix}",
-        "--darwin-deployment-version-osx=#{MacOS.version}",
-        "--build-jobs=#{ENV.make_jobs}"
+        "--build-jobs=#{ENV.make_jobs}",
+        ("--darwin-deployment-version-osx=#{MacOS.version}" if OS.mac?)
```

Comment:
Please move this line back up where it was between `--install-prefix` and `--build-jobs`.